torch/csrc/deploy/interpreter/interpreter_impl.cpp

```diff
@@ -0,0 +1,455 @@
+#include <dlfcn.h>
+
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+#include <torch/csrc/deploy/interpreter/interpreter_impl.h>
+#include <iostream>
+
+#include <assert.h>
+#include <pybind11/embed.h>
+#include <stdio.h>
+#include <torch/csrc/autograd/generated/variable_factories.h>
+#include <torch/csrc/jit/python/pybind_utils.h>
+#include <iostream>
+#include <map>
+#include <thread>
+
+#include <fmt/format.h>
+
+namespace py = pybind11;
+using namespace py::literals;
+
+// TODO this should come from cmake
+#define DEBUG 1
```

Comment:
do you just want to have debug mode? in this case `NDEBUG` (for Release) is defined correctly (it's actually a compiler de-facto standard while `DEBUG` is not)